packages/react-scripts/template/src/components/example/Logo.js

```diff
@@ -42,4 +43,11 @@ const Logo = ({ color = 'black', animationDuration = 20 }) => {
   )
 }
 
+Logo.propTypes = {
+  /** Used to set the color of the logo. */
+  color: PropTypes.string.isRequired,
+  /** Used to set the duration of animation of the logo. */
+  animationDuration: PropTypes.number.isRequired,
```

Comment:
not required

Reply:
`.isRequired` is not required or PropTypes for Logo is not required?

Reply:
the .isRequired